README.md

```diff
@@ -46,4 +46,4 @@ By showing critical UI in the body of the content, many can get their basic blog
 
 When the Post Settings sidebar is open — which it is by default — you are essentially in advanced layout mode. By default, you'll see all your metaboxes right there.
 
-Every block can be _inspected_ by clicking it. And every block has advanced layout options available in the inspector; text might have drop-cap, image might have fixed position scrolling. As such, block attributes fall in two camps — the most important ones available right on the block, advanced ones living in the sidebar inspector.
+Every block can be _inspected_ by clicking it. And every block has advanced layout options available in the inspector; text might have drop-cap, the image might have fixed position scrolling. As such, block attributes fall into two camps — the most important ones available right on the block, advanced ones living in the sidebar inspector.
```

Comment:
"the image" - `image` refers to a specific block type, same as `text`. We can put backticks `\`` around them but they are correct as currently written.

"into" - :+1: